js/app.js

```diff
@@ -1,23 +1,58 @@
 "use strict";
-
+var currentUser = {};
```

Comment:
oh, yeah, that's unnecessary, and should be removed. want to talk about how to implement something that shares `currentUser` globally or something. I think a service is the right move here?